run

```diff
@@ -18,6 +18,8 @@ EXE_PATH=$(find "${SCRIPT_PATH}"/target/java-compiler-*-with-dependencies.jar) |
 # ensure that jFirm can find libfirm library
 export LD_LIBRARY_PATH="${SCRIPT_PATH}/target/lib/"
 
+export STD_LIBRARY_PATH="${SCRIPT_PATH}/lib/stdlib.c"
```

Comment:
I think this should also be included in the build script, in case unit tests want to use it

Reply:
I have added the environment variable to the build script. It's going to be difficult to test this in unit tests though. We would need to link the assembly generated by jFirm to the library (`gcc` call in the CLI).